docs/2.0/docs/patcher/tutorials/applying-first-patch.md

```diff
@@ -1,3 +1,161 @@
 # Applying your first patch
+import CustomizableValue from '/src/components/CustomizableValue';
 
-This page is under construction.
\ No newline at end of file
+## Prerequisites
+* You have familiarity with Terraform or OpenTofu
+* You have [patcher installed](/2.0/docs/patcher/installation/) either locally or as a GitHub Actions Workflow in your repository
+
+## Overview
+
+In this tutorial we will walk you through the following steps:
+
+**Using Patcher Locally**:
+* Creating a new [infrastructure Unit](https://terragrunt.gruntwork.io/docs/getting-started/terminology/#unit) referencing a sample OpenTofu module, intentionally specifying an outdated version
+* Running patcher against that module and understanding the outputs / options Patcher provides.
+* Instructing patcher to apply the patch
+* Creating a commit incorporating the patch
+
+**Using Patcher as a GitHub Actions Workflow**:
+* Creating a new [infrastructure Unit](https://terragrunt.gruntwork.io/docs/getting-started/terminology/#unit) referencing a sample OpenTofu module, intentionally specifying an outdated version
+* Pushing that unit up to your remote repository in GitHub
+* Triggering patcher to detect the out of date unit and apply the patch
+* Reviewing the pull request created by Patcher and merging the changes.
+
+### The `patcher-test` Module
+
+In this tutorial we'll use the `patcher-test` module from the `gruntwork-io/terraform-aws-utilities` repository. This module has a version `v0.10.3` that we'll intentionally specify as outdated in our infrastructure unit, and have patcher update to v0.10.4 which includes a demonstration breaking change.
```

Comment:
nit: would you be opposed to having this be `v0.11.0` to reflect semantic versioning's concept of a breaking change?

Reply:
Not opposed, it's just legwork to change the actual repos/make those releases as I don't think they exist in this state at the moment.

Reply:
ah - that helps to know that this example is one reflected literally in the repo. thank you!

Reply:
Correct -- a user has to be able to follow these steps exactly and get the same results